esphome/components/dsmr/text_sensor.py

```diff
@@ -22,6 +22,7 @@
         cv.Optional("water_equipment_id"): text_sensor.text_sensor_schema(),
         cv.Optional("sub_equipment_id"): text_sensor.text_sensor_schema(),
         cv.Optional("gas_delivered_text"): text_sensor.text_sensor_schema(),
+        cv.Optional("telegram"): text_sensor.text_sensor_schema(),
```

Comment:
This wont pass CI, but is how you do it.
```suggestion
        cv.Optional("telegram"): text_sensor.text_sensor_schema().extend({cv.Optional(CONF_INTERNAL, default=True): cv.boolean),
```

Reply:
Thanks @jesserockz !
 It takes a while but now I understand how it works. I've tested you're suggested change and it of course it works. The documentation will be updated as well. (strange enough the CI pytest on linux 3.11 fails, but on the other mac/windows they succeed trying to reproduce and fix)